CONTRIBUTING.md

```diff
@@ -150,6 +150,27 @@ We're currently experimenting with hosting the majority of lodestar packages and
   - Error: One or more main functionalities are not working, preventing some functions from working properly.
   - Fatal: One or more main functionalities are not working and preventing the application from fulfilling its duties.
 
+## Logging policy
+
+### Logging Levels
+
+- `error`: Critical issues that prevent the application from functioning correctly or cause significant disruption to users. Examples include failed network connections, crashes, or data corruption.
+- `warn`: Situations that may lead to critical issues if not addressed but do not prevent the application from functioning. Examples include configuration issues, deprecated features, or temporary network disruptions.
+- `info`: General sporadic informational about the node's state. Examples include initialization messages, infrequent periodic status updates, or high-level progress reports.
+- `verbose`: High-level state changes to internal modules. Examples include chain and state updates.
+- `debug`: Detailed diagnostic information that can help developers or users troubleshoot specific issues. Examples include individual request logs for every REST API, networking interactions, or internal components status changes.
+
+### Guideliness
+
+1. Always use the appropriate log level as defined above. Log messages should be clear and concise, providing enough information to understand the context and severity of the issue.
```

Comment:
I think `1.` and `2.` are essentially saying very similar things. I would combine them to say

```suggestion
1. Logs should state clearly and concisely only that which is required to understand / troubleshoot an issue. Including any error codes, stack traces, debug logs or environment variables which are applicable.
```

Reply:
Simplified 1 and 2